tutoraspects/templates/openedx-assets/assets/dashboards/Instructor_dashboard.yaml

```diff
@@ -216,10 +218,11 @@ metadata:
     - NATIVE_FILTER-Vx7HxG8_7
     - NATIVE_FILTER-XPuiTOej4
     - NATIVE_FILTER-E6-vOpjZv
-    chartsInScope: []
+    chartsInScope:
+    - 52
```

Comment:
Is this the only chart that is affected by this filter? I think this filter has a wider scope

Reply:
That's interesting. The Superset UI is showing that this filter applies to all four charts in the "problem performance" tab. I'll take a look.

Reply:
Let me know when you get this sorted, we should merge this before my big refactor PR hits otherwise it will be a nightmare to rebase

Reply:
I'm not sure why the config looks like this, but I've checked the queries generated for the instructor dashboard they're using the correct filters.